pkg/controller/kogitoinfra/kafka.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kogitoinfra
+
+import (
+	"fmt"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	kafkabetav1 "github.com/kiegroup/kogito-cloud-operator/pkg/apis/kafka/v1beta1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sort"
+	"strings"
+	"time"
+)
+
+const (
+	enableEventsEnvKey = "ENABLE_EVENTS"
+
+	// QuarkusKafkaBootstrapAppProp quarkus application property for setting kafka server
+	QuarkusKafkaBootstrapAppProp = "kafka.bootstrap.servers"
+
+	// springKafkaBootstrapAppProp spring boot application property for setting kafka server
+	springKafkaBootstrapAppProp = "spring.kafka.bootstrap-servers"
+
+	// KafkaInstanceName is the default name for the Kafka cluster managed by KogitoInfra
+	KafkaInstanceName = "kogito-kafka"
```

Comment:
moved to infrastructure pkg: https://github.com/kiegroup/kogito-cloud-operator/blob/master/pkg/infrastructure/kafka.go#L42

Reply:
same as previous